src/routes/PublicLayoutRouter/PublicLayoutRouter.scss

```diff
@@ -74,9 +74,6 @@
 
 .non-map-view {
   padding-bottom: 100px;
-}
-
-.non-map-view > {
```

Comment:
what was this about?

Reply:
I didn't actively change this but also, the prior code doesn't seem to make sense...

What would `.non-map-view > {` even do differently from `.non-map-view {`? It would need another selector, right?